webapp/client/src/components/ShareIcons.test.js

```diff
@@ -26,3 +27,32 @@ describe("ShareIcons no plausible", () => {
     expect(screen.queryByLabelText("whatsapp")).not.toBeInTheDocument();
   });
 });
+
+describe("ShareIcons with plausible", () => {
+  const MOCK_PROPS_PLAUSIBLE = {
+    promoteUrl: "www.google.com",
+    shareText: "Hey such an amazing search machine",
+    mailSubject: "Did you know it already?",
+    sourcePage: "ShareIcons.test",
+    plausibleDomain: "http://localhost:3000",
+  };
+
+  beforeEach(() => {
+    window.plausible = jest.fn().mockReturnValue({ plausible: jest.fn() });
+    render(<ShareIcons {...MOCK_PROPS_PLAUSIBLE} />);
+  });
+
+  it("should add plausible goal for facebook icon", () => {
+    fireEvent.click(screen.getByLabelText("facebook"));
```

Comment:
You could use userEvent.click because this will not only fire the click Event but simulate a real click (MouseDown...) . Have a look at e.g. the  FormFieldCheckBox.test.js

Reply:
Done :)